docs/src/theme/Blog/Components/Author/index.tsx

```diff
@@ -0,0 +1,93 @@
+import Link, { type Props as LinkProps } from '@docusaurus/Link';
```

Comment:
Had to add this to edit the styles.module.css as it is currently broken in the latest version...

Reply:
Otherwise this component is unmodified.